src/Idle.DataAccess/Idle.DataAccess.csproj

```diff
@@ -12,7 +12,7 @@
   <ItemGroup>
     <PackageReference Include="Sharpnado.Shadows" Version="1.2.0" />
     <PackageReference Include="Sharpnado.Tabs" Version="2.1.0" />
-    <PackageReference Include="sqlite-net-pcl" Version="1.7.335" />
+    <PackageReference Include="sqlite-net-pcl" Version="1.6.292" />
```

Comment:
Didn't check this yet. But this is not important currently. Can do this later ad's well